docs/API.md

```diff
@@ -1,47 +1,4 @@
 # Component API
 
 * [Component props](APIProps.md)
-* [Component methods](APIMethods.md)
-
-## Props quick reference
```

Comment:
I think you might want to add a link to types/index.d.ts here? People might not know to look for that immediately